shared-module/bitmaptools/__init__.c

```diff
@@ -252,6 +251,148 @@ void common_hal_bitmaptools_fill_region(displayio_bitmap_t *destination,
     }
 }
 
+void common_hal_bitmaptools_boundary_fill(displayio_bitmap_t *destination,
+    int16_t x, int16_t y,
+    uint32_t fill_color_value, uint32_t replaced_color_value) {
+
+    if (fill_color_value == replaced_color_value) {
+        // There is nothing to do
+        return;
+    }
+
+
+
+
+    uint32_t current_point_color_value;
+
+    // the list of points that we'll check
+    mp_obj_t fill_area = mp_obj_new_list(0, NULL);
+
+    // first point is the one user passed in
+    mp_obj_t point[] = { mp_obj_new_int(x), mp_obj_new_int(y) };
+    mp_obj_list_append(
+        fill_area,
+        mp_obj_new_tuple(2, point)
+        );
+
+    int16_t minx = x;
+    int16_t miny = x;
```

Comment:
Shouldn't miny = y and maxx  = x respectively?
With the current setting say a 1 point fill at (5,10) would mark the dirty area as (5,5) to (10,10) instead of just (5,10) to (5,10). Or another way if x starts at 5 but moves 2 wide to 7, the maxx is still 10 which is larger then the changes area.

Reply:
whoops, yes each should be their own respective values. Thank you

Reply:
fixed this with latest commit